docker/openproblems-python-extras/requirements.txt

```diff
@@ -1 +1,2 @@
 git+https://github.com/KrishnaswamyLab/harmonic-alignment#subdirectory=python
+pybedtools
```

Comment:
pyensembl should be added to this docker image

Reply:
it is added into the docker image.